packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -0,0 +1,393 @@
+<template>
```

Comment:
Is it worth splitting some of the components out into smaller ones? Would mean you could move some of the logic into those smaller components too. I think these are the main components? 

1. `c-cookieBanner-CTA.vue`
2. `legacyBanner.vue`
3. `cookieBanner.vue` (Would just be the main component or wrapper you have already in this file.)

Reply:
I have moved the legacy banner to a component, as this will make it easier to remove in a couple of weeks time. However, I don't think it's worth moving the cookie banner or buttons out to components, as the cookie banner is just a heading and three paragraphs, and the buttons are already imported from f-button - just seems like an unnecessary layer of complexity (KISS).